patches/chrome-browser-resources-signin-profile_card.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/signin/profile_picker/profile_card.html b/chrome/browser/resources/signin/profile_picker/profile_card.html
+index 2c239a2a7744a..06facf8a74833 100644
+--- a/chrome/browser/resources/signin/profile_picker/profile_card.html
```

Comment:
I am not sure what happened here, but the patch file names are incorrect in this PR. Should be 
```
patches/chrome-browser-resources-signin-profile_picker-profile_card.html.patch
patches/chrome-browser-resources-signin-profile_picker-profile_picker_main_view.html.patch
```
(missing `profile_picker` after `signin`).

Reply:
https://github.com/brave/brave-core/pull/22936/commits/3208a29e934d30f666d5291d4745a83446dabb2e - fix in cr125 branch

Reply:
Nice catch - fix is https://github.com/brave/brave-core/pull/23247